Marlin/language_de.h

```diff
@@ -153,6 +159,12 @@
 #define MSG_ERR_MINTEMP                     LCD_STR_THERMOMETER " UNTERSCHRITTEN"
 #define MSG_ERR_MAXTEMP_BED                 "BETT " LCD_STR_THERMOMETER " ÜBERSCHRITTEN"
 #define MSG_ERR_MINTEMP_BED                 "BETT " LCD_STR_THERMOMETER " UNTERSCHRITTEN"
+#define MSG_ERR_Z_HOMING                    "G28 Z Forbidden"
```

Comment:
That seems to be a good idea. :) But then there is maybe a "bug" in the code:

`// Disallow Z homing if X or Y are unknown`
    `if (!axis_known_position[X_AXIS] || !axis_known_position[Y_AXIS]) {`
      `LCD_MESSAGEPGM(MSG_ERR_Z_HOMING);`
      `SERIAL_ECHO_START;`
      `SERIAL_ECHOLNPGM(MSG_ERR_Z_HOMING);`
      `return;`
    `}`
[https://github.com/MarlinFirmware/Marlin/blob/RCBugFix/Marlin/Marlin_main.cpp#L2746](Marlin_main.cpp)

So that string is used for both